lib/xcode/install.rb

```diff
@@ -80,6 +96,31 @@ def installed_versions
       end
     end
 
+    # Returns an array of `XcodeInstall::Xcode`
+    #   <XcodeInstall::Xcode:0x007fa1d451c390
+    #     @date_modified=2015,
+    #     @name="6.4",
+    #     @path="/Developer_Tools/Xcode_6.4/Xcode_6.4.dmg",
+    #     @url=
+    #      "https://developer.apple.com/devcenter/download.action?path=/Developer_Tools/Xcode_6.4/Xcode_6.4.dmg",
+    #     @version=Gem::Version.new("6.4")>,
+    #
+    # the resulting list is sorted with the most recent release as first element
+    def seedlist
+      @xcodes = Marshal.load(File.read(LIST_FILE)) if LIST_FILE.exist? && xcodes.nil?
+      all_xcodes = (xcodes || fetch_seedlist)
+
+      # We have to set the `installed` value here, as we might still use
+      # the cached list of available Xcode versions, but have a new Xcode
+      # installed in the mean-time
+      cached_installed_versions = installed_versions.map(&:bundle_version)
+      all_xcodes.each do |current_xcode|
+        current_xcode.installed = cached_installed_versions.include?(current_xcode.version)
+      end
+
+      all_xcodes.sort_by(&:version)
```

Comment:
explicit return preferred

Reply:
![tableflip](https://user-images.githubusercontent.com/1304821/39828452-e4f500b4-536f-11e8-9eb0-d60d9740f9db.gif)

Reply:
`all_xcodes` is an array of `Xcode` object, whose `:version` attribute is of type `Gem::Version`, resulting in everything to be properly sorted 👍